lighthouse-core/fraggle-rock/gather/session.js

```diff
@@ -73,10 +73,18 @@ class ProtocolSession {
    * Bind to our custom event that fires for *any* protocol event.
    * @param {(payload: LH.Protocol.RawEventMessage) => void} callback
    */
-  onAnyProtocolMessage(callback) {
+  addProtocolMessageListener(callback) {
     this._session.on('*', /** @type {*} */ (callback));
   }
 
+  /**
+   * Unbind to our custom event that fires for *any* protocol event.
+   * @param {(payload: LH.Protocol.RawEventMessage) => void} callback
+   */
+  removeProtocolMessageListener(callback) {
+    this._session.off('*', /** @type {*} */ (callback));
```

Comment:
Not sure how accurate these codecov warnings are. Looks like they just miss by a few lines.

Could you add a test case for this function?